com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryHapticSocketEditor.cs

```diff
@@ -422,6 +438,12 @@ public class VRCFuryHapticPlaySocketEditor : UnityEditor.Editor {
             return ShouldProbablyHaveTouchZone(socket);
         }
 
+        public static bool ShouldProbablyBeReversible(VRCFuryHapticSocket socket) {
+            if (HapticUtils.IsChildOfBone(socket.owner(), HumanBodyBones.LeftHand)) return true;

```

Comment:
I don't think we should do a guess here. The vast majority of rings should be double sided, including ones that don't wind up on hands. I don't think it's unreasonable to make people select single sided if they choose for some specific use case.